example/src/main/java/com/thesurix/example/gesturerecycler/adapter/HeaderViewHolder.java

```diff
@@ -1,25 +1,35 @@
 package com.thesurix.example.gesturerecycler.adapter;
 
-import com.thesurix.example.gesturerecycler.R;
-import com.thesurix.gesturerecycler.GestureViewHolder;
-
+import android.support.annotation.NonNull;
 import android.view.View;
 import android.widget.TextView;
 
+import com.thesurix.example.gesturerecycler.R;
+import com.thesurix.example.gesturerecycler.model.MonthHeader;
+import com.thesurix.example.gesturerecycler.model.MonthItem;
+import com.thesurix.gesturerecycler.GestureViewHolder;
+
 import butterknife.BindView;
 import butterknife.ButterKnife;
 
-
-public class HeaderViewHolder extends GestureViewHolder {
+public class HeaderViewHolder extends GestureViewHolder<MonthItem> {
 
     @BindView(R.id.header_text)
     TextView mHeaderText;
 
-    public HeaderViewHolder(final View view) {
+    public HeaderViewHolder(@NonNull View view) {
         super(view);
         ButterKnife.bind(this, view);
     }
 
+    @Override
+    public void bindHolder(MonthItem monthItem) {
+        if (monthItem.getType() == MonthItem.MonthItemType.MONTH) {
```

Comment:
Please check out examples, I can not see headers.

Reply:
@thesurix Yes, my bad. Fixed this